web-registry/src/pages/Marketplace/Storefront/Storefront.normalizer.ts

```diff
@@ -58,7 +58,9 @@ export const normalizeProjectsInfosByHandleMap = ({
 
     projectsMap.set(project?.id, {
       name: project.metadata?.['schema:name'] || project.id,
+      // TODO: why is this being called classIdName?
       classIdName: creditClassSanity?.nameRaw ?? project?.classId,
+      // TODO: why is this being called classIdUrl?
```

Comment:
Good catch, definitely something weird here.

Reply:
looking at its usage, I think it's because it's meant to be used within URLs but I guess `classId` would just be ok too...

Reply:
did a find and replace in this commit: 5bf681b055666e18ada4556ed072002aa37f0040

Reply:
thanks, resolving